nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3256,9 +3361,9 @@ stream {
 
 image:s2s-rproxy-portnumber.svg["Port number to Node mapping"]
```

Comment:
Hi @zenfenan,
Can you clarify what you mean by "aren't loading properly"?  I'm trying to understand what is broken since the screenshots are properly displayed in the generated docs.

Reply:
Does the screenshots load for you? It doesn't for me. Tried across browsers and also confirmed that the URL is different. 
![pics not loading](https://user-images.githubusercontent.com/23693101/43779029-e9892ece-9a74-11e8-8014-edec377b122c.PNG)

Reply:
While editing a doc (administration-guide.adoc, overview.adoc, etc.), the screenshots won't load unless the image files are in the same directory as the doc.  But, once the docs are build the screenshots will be referenced properly:

![generated_doc](https://user-images.githubusercontent.com/18287533/43781527-165ebef4-9a2b-11e8-83d3-0df58973005e.jpg)

Reply:
Got it but I still prefer the image being loaded on the GitHub md files though. Anyways, I'm good. I'll go ahead and merge it once the broken link for BCrypt is fixed.

Reply:
Thanks @zenfenan .  Added a commit with that fix.